### DISCUSSION TITLE
Does this plugin sync collections also?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [statamic-rad-pack/shopify#22] [Feature Requests]
Your plugin looks awesome.  Does it also synchronize with Shopify collection objects?

Being that their collection feature is pretty powerful, that would be very nice to have (if it's not included already)

Cheers

**Comment by jackabox** [comment]
It doesn't currently as I'm not sure how best to represent this under Statamic.

At the moment it does pull in **product type, vendors, and tags** as taxonomies.

I don't think this would be too hard to pull in as long as it's called after the products are imported. We should be able to bind the entries based on the data returned and this could be totally optional. This would probably have to be a **collection** within Statamic which could get confusing on the wording.

How'd you envision using them within your site?

**Reply by dungle-scrubs** [reply]
Nice, in my mind **collection** makes sense as the term as it lines up with the name of the shopify object itself. 

I envision it as query for a group of product objects that can be created and managed by the shop owner. Since that's one of the very nice features of Shopify - being able to create collections manually or dynamically in many different ways - it makes more sense to me to not need to do that at the CMS level.

**Reply by jackabox** [reply]
Makes sense, leave this with me and I'll see if I can do something nice about it 👍🏻

**Comment by sebszocinski** [comment]
Plus one over here, I think collections are important purely because of the powerful options Shopify gives you in terms of automated / dynamic collections based on things like title, types, tags, vendors etc.

I feel like it would be safe to set them up as terms of a taxonomy however since there isn't any nesting capability in Shopify Collections so we don't need that support.

**Comment by austriker27** [comment]
I would also love to see this included! 

Here's the info I'd use from a collection: collection title, description, image and products.

Ideally, I would manage the collection content and filtering of data ("Conditions") on shopify's end.

**Comment by edalzell** [comment]
I think this should be a Taxonomy (so each Shopify Collection is a `term`). Makes for easy grouping/filtering in Statamic.

**Comment by jackabox** [comment]
Collections are imported now, so closing this. :)
